clients/instance/ibm-pi-instance.go

```diff
@@ -55,8 +55,8 @@ func (f *IBMPIInstanceClient) GetAll() (*models.PVMInstances, error) {
 // Create an Instance
 func (f *IBMPIInstanceClient) Create(body *models.PVMInstanceCreate) (*models.PVMInstanceList, error) {
 	// Check for satellite differences in this endpoint
-	if f.session.IsOnPrem() && (body.SoftwareLicenses != nil || body.SharedProcessorPool != "") {
-		return nil, fmt.Errorf("software licenses and shared processor pool parameters are not supported in satellite location, check documentation")
+	if f.session.IsOnPrem() && (body.DeploymentTarget != nil || body.SharedProcessorPool != "") {
```

Comment:
I thought `ibmi` and `spp` were go for starts q2.  Do we still need a check it here?

Reply:
You are right, I forgot to remove the spp check here.